### ISSUE TITLE
Single v Multi Geometries

### ISSUE THREAD

**Issue by philipbaileynar** [namc-utah/namc-bug-db#57]
Does anyone see a problem storing sites as single points and their catchments as single contiguous polygons?

In other words, can sites ever have two or more locations? (I think this one is obviously "no", but I just want to confirm.)

And can catchments be multiple discontinuous polygons? This one is up for debate. Perhaps a sections of a catchment "go to ground" and then resurface downstream requiring two polygons to be captured? We can easily use multi-
polygons is needed. Please confirm.

![polygon2](https://user-images.githubusercontent.com/8645379/106669256-3ecfbe80-6560-11eb-81a5-6010c65f65a2.jpg)

**Comment by courtwrightj01** [comment]
Sites should be single points. I assume we will want to build in flexibility for the polygon scenario you have below though. I think allowing for multiple discontinuous polygons would be safest

**Comment by alherca73** [comment]
I concur with Jennifer, allowing multipolygons is a good idea. As I get polygons from StreamStats or from ArcMap processes I've found both (polygons / multi-polygons). Most times I can extract predictors without issues (even in simple features with both types). Most times that I've experienced issues trying to extract zonal stats, is because of multi-polygons. Sometimes I can fix this with something like st_makevalid. Several times I've had to simplify or force one type of geometry (i.e. cast multipolygons to polygons or viceversa).
